### DISCUSSION TITLE
Virtual Desktop Preserve Taskbar Order v1.0.4 is not working for me

### DISCUSSION THREAD

**Discussion by jimday** [ramensoftware/windhawk-mods#995] [Q&A]
I installed Windhawk v1.5.1 with the above mentioned mod in Win11 23H2 b22631.4169, but it's not working for me, it won't preserve the taskbar software icon orders after switching virtual desktops. Anybody can help with this issue, please?

Below are the detailed debug logs when trying the mod out:
https://justpaste.it/cdtiy

Thanks!

**Comment by m417z** [comment]
The mod has some limitations which are described in detail in my comment here: https://github.com/ramensoftware/windhawk-mods/issues/458. Some limitations involve pinned items and windows which are shown in all/several monitors. Please try to describe in which cases the mod doesn't work for you, and let's understand whether the issue you see is a duplicate of #458.

See also: https://github.com/ramensoftware/windhawk-mods/issues/889.
